docs/api/ref/DesktopAgent.md

```diff
@@ -67,7 +67,7 @@ Context broadcasts are only received from apps that are joined to the same User
 
 Optional metadata about each context message received, including the app that originated the message, SHOULD be provided by the desktop agent implementation.
 
-#### Examples
+**Examples:**
```

Comment:
This change was made by a markdown linting tool. It'll appear in a few different PRs. I'll try and find time to run it over the things in a separate PR so that it drops out of this one once merged.

Reply:
Should the change be made though just because the markdown linter tool says so though? Linter tools aren't always right or applicable. Just tell the linter tool to ignore MD docs 🤷

Reply:
In this case, I think its right on the formatting change, these are not navigable headings, are repeated throughout the doc and look the same after the tweak to the markdown.

Reply:
Raised a separate PR that corrects all the linting errors and spelling errors through all the docs, plus a handful of formatting issues I spotted while doing it. Once that's merged I'll rebase this branch to drop out the same changes (and try to figure what other open PRs have them and do the same, I believe its several).

Need a review to complete: https://github.com/finos/FDC3/pull/1002

Reply:
The majoriting of such changes have been dropped out of the PR (were made in a separate PR now merged to master which this PR was rebased on top of). There may be one or two minor ones left at most.